tests/test_commit_command.rs

```diff
@@ -54,6 +54,25 @@ fn test_commit_with_editor() {
         std::fs::read_to_string(test_env.env_root().join("editor0")).unwrap(), @r###"
     initial
 
+    JJ: This part contains the following changes:
```

Comment:
"This part" sounds weird when we're not talking about splitting a commit. I think it's clear enough to use "This commit" for all cases (`commit`, `split`, etc.). What do you think?

Reply:
Sounds good, done.